tests/drivers/candev/Makefile.board.dep

```diff
@@ -7,6 +7,10 @@ CAN_DRIVER ?= MCP2515
 
 ifeq ($(CAN_DRIVER), PERIPH_CAN)
   FEATURES_REQUIRED += periph_can
+  ifeq ($(BOARD), same54-xpro)
+    CFLAGS += "-DCANDEV_SAMD5X_DEFAULT_STD_FILTER_NUM=5"
+    CFLAGS += "-DCANDEV_SAMD5X_DEFAULT_EXT_FILTER_NUM=5"
+  endif
```

Comment:
Why are the defaults only set that way for `same54-xpro`?
There can be other boards with that CAN peripheral

Reply:
I saw in boards definition that it's the only board using the SAMD/E5x cpu, so I limited these defaults to this board. But you're right, I can leave that out